smf/manifests/buckets-mdplacement.xml.in

```diff
@@ -10,8 +10,8 @@
     Copyright 2019 Joyent, Inc.
 -->
 
-<service_bundle type="manifest" name="-electric-boray">
-    <service name="manta/application/electric-boray" type="service" version="1">
+<service_bundle type="manifest" name="-buckets-mdplacement">
+    <service name="smartdc/application/buckets-mdplacement" type="service" version="1">
```

Comment:
Not "manta/..." still here?  Was that intentional?

Reply:
Not intentional at all. Nice catch!

Reply:
Fixed